cad/LibreCAD/Portfile

```diff
@@ -30,7 +30,11 @@ if {${subport} eq "${name}"} {
 
     checksums           rmd160  ad84d422534a69c49a85017e0f04ba050115fb89 \
                         sha256  c23e6a1ccd0368c089e74772c02a420a5b475fdeb205ae6be39a93b3abeda965 \
-                        size    13578523
+        size    13578523
```

Comment:
Is there a reason for the indentation change? If not, maybe you can fix that?

Reply:
Just emacs auto-indenting incorrectly. I noticed after I submitted the pull request. Is there a way to fix it without redoing the entire pull request process?

Reply:
Of course. Do this:

1. `git reset --soft HEAD^` — this will "uncommit" your change.
2. edit and fix the problem, and re-commit the fix.
3. force push you change with `git push origin +librecad_qt511`

Reply:
What I would usually do is the following:
```
git checkout librecad_qt511
# edit the file
git add cad/LibreCAD/Portfile
git commit --amend
git push -f <your-repo>
```
The `git commit --amend` will do a change to the last commit, while `git reset --soft HEAD^` will first undo the last commit and then you would make a new one. Resetting is usually more helpful when you added unwanted changes (and unwanted files) to the commit, while `git commit --amend` is a shortcut for trivial changes or when you want to change the commit message.

The important part is to force push the changes to the branch.

Reply:
OK, I followed mojca's suggestions for fixing this one-liner. All should be good now. Thanks.